packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -246,6 +247,9 @@ export interface ISummaryRuntimeOptions {
      * THis defaults to false (disabled) and must be explicitly set to true to enable.
      */
     summarizerClientElection?: boolean;
+
+    /** Options that control the running summarizer behavior. */
+    summarizerOptions?: Readonly<Partial<ISummarizerOptions>>;
```

Comment:
`ISummarizerOptions` is easy to confuse with `ISummaryRuntimeOptions`. I think we should choose a name which makes it more explicit what it is for.

Reply:
Well in this case I think `ISummarizerOptions` is accurate already. I recognize they are confusing, but not sure how to disambiguate these.. open to suggestions.

Reply:
I guess this is fine. We can rename it later if needed.